src/Tests/UnitTests.proj

```diff
@@ -61,6 +61,8 @@
 
       <Testpackages Include="$(ArtifactsTmpDir)testpackages\**\*.*"/>
 
+      <!-- include Container artifacts for running test in Helix -->
```

Comment:
Pending to implement. Refer to how to deal with `<Testpackages Include="$(ArtifactsTmpDir)testpackages\**\*.*"/>`.